container.go

```diff
@@ -392,7 +392,10 @@ func (c *ContainerRequest) BuildOptions() (types.ImageBuildOptions, error) {
 	}
 
 	if !c.ShouldKeepBuiltImage() {
-		buildOptions.Labels = core.DefaultLabels(core.SessionID())
+		err := core.MergeCustomLabels(buildOptions.Labels, core.DefaultLabels(core.SessionID()))
+		if err != nil {
+			panic(fmt.Errorf("Unable to merge custom labels"))
```

Comment:
We probably don't want to panic here, but pass the error to the caller. In any case, see my comment in https://github.com/testcontainers/testcontainers-go/pull/2637/files#r1673820440